snoop/data/models.py

```diff
@@ -882,3 +882,32 @@ def __str__(self):
     class Meta:
 
         verbose_name_plural = 'statistics'
+
+
+class Thumbnail(models.Model):
+    """Database model for storing the Thumbnail corresponding to a Digest.
+    """
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=['blob', 'size'], name='unique_size')
+        ]
+
+    class Size(models.IntegerChoices):
+        SMALL = 100
+        MEDIUM = 200
+        LARGE = 400
+
+    blob = models.ForeignKey(
+        Blob,
+        on_delete=models.CASCADE,
+        related_name='file_blob',
```

Comment:
Actually, the related name is the name you'd use on the referenced table to get back to this one. So the related names here are incorrect, since the objects you get are going to be Thumbnail, not Blob.


 Since we don't want to get from any `Blob` back to the thumbnail by using some related name directly, let's deactivate it by setting it to `+`.

https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.ForeignKey.related_name